uc-crux-llvm/src/UCCrux/LLVM/Stats.hs

```diff
@@ -54,10 +55,11 @@ frequencies = foldr (\tag mp -> Map.insertWith (+) tag 1 mp) Map.empty
 
 getStats :: BugfindingResult m arch argTypes -> Stats
 getStats result =
-  let (missingAnns, failedAsserts, unimplementeds, unclass, unfixed, unfixable) = partitionUncertainty (uncertainResults result)
+  let (missingAnns, failedAsserts, unimplementeds, unclass, unfixed, unfixable, timeouts') = partitionUncertainty (uncertainResults result)
```

Comment:
That internal record data structure would help here as well.

Reply:
https://github.com/GaloisInc/crucible/issues/691